rskj-core/src/main/java/co/rsk/peg/BridgeStorageProvider.java

```diff
@@ -884,6 +890,53 @@ public void saveReceiveHeadersLastTimestamp() {
         }
     }
 
+    public Optional<Long> getNextUtxoExpirationCheckpointHeight() {
+        if (!activations.isActive(RSKIP264)) {
+            return Optional.empty();
+        }
+
+        if (nextUtxoExpirationCheckpointHeight == null) {
+            nextUtxoExpirationCheckpointHeight = safeGetFromRepository(NEXT_UTXO_EXPIRATION_CHECKPOINT_HEIGHT, BridgeSerializationUtils::deserializeOptionalLong).orElse(0L);
+        }
+
+        return Optional.of(nextUtxoExpirationCheckpointHeight);
+    }
+
+    public void setNextUtxoExpirationCheckpointHeight(Long nextUtxoExpirationCheckpointHeight) {
+        if (activations.isActive(RSKIP264) && nextUtxoExpirationCheckpointHeight != null) {
```

Comment:
I suggest changing the type to long to discard possible null values, and instead check that the value is always positive.

Reply:
Resolved